src/mpn_extras/profile/p-mulhigh_basecase.c

```diff
@@ -12,21 +12,80 @@
 #include "mpn_extras.h"
 #include "profiler.h"
 
+#if !FLINT_HAVE_ADX
+# error
```

Comment:
BTW, better to put in a dummy main function here; otherwise ``make profile`` breaks on other configurations.

Reply:
True.